api.planx.uk/modules/slack/index.test.ts

```diff
@@ -12,14 +12,16 @@ const invalidBody = {
   wrong: "message",
 };
 
-const mockSend = jest.fn();
-jest.mock<typeof SlackNotify>("slack-notify", () =>
```

Comment:
tsc/vitest seem smart enough not to need the type declaration anymore 🤔

Reply:
Ah that's awesome because this step always felt awkward and brittle in jest imho 🙌